### PR TITLE
fix support calling sqs from api ws gw on disconnect endpoint

### PR DESCRIPTION
**Pull request by calvernaz**
## Motivation

The user issue was related to a call to the "$disconnect" route and not triggering the SQS backend integration. The problem is we didn't support WS+SQS integration until now. This PR fixes that.

### CODE REVIEW

File: localstack/services/apigateway/integration.py [hunk 1]
```diff
@@ -682,12 +698,17 @@ def invoke(self, invocation_context: ApiInvocationContext):
         if "Accept" not in headers:
             headers["Accept"] = "application/json"
 
-        template = integration.get("requestTemplates").get(APPLICATION_JSON)
-        if "GetQueueUrl" in template or "CreateQueue" in template:
+        if invocation_context.is_websocket_request():
+            template_key = self.render_template_selection_expression(invocation_context)
+            payload = self.request_templates.render(invocation_context, template_key)
```
Comment (bentsku) [inline localstack/services/apigateway/integration.py:703]: I think this should be the following, from what I've seen in our call yesterday:
```suggestion
            payload = self.render_template_selection_expression(invocation_context)
```

File: localstack/services/apigateway/integration.py [hunk 2]
```diff
@@ -466,7 +483,6 @@ def invoke(self, invocation_context: ApiInvocationContext):
             # integration type "AWS" is only supported for WebSocket APIs and REST
             # API (v1), but the template selection expression is only supported for
             # Websockets
-            template_key = None
             if invocation_context.is_websocket_request():
                 template_key = invocation_context.integration.get(
                     "TemplateSelectionExpression", "$default"
```
Comment (bentsku) [inline localstack/services/apigateway/integration.py:488]: q: should this also be replace with `render_template_selection_expression`?

File: localstack/services/apigateway/integration.py [hunk 3]
```diff
@@ -119,6 +119,23 @@ def apply_response_parameters(
                 response.headers[header_name] = value.strip("'")
         return response
 
+    @classmethod
+    def render_template_selection_expression(cls, invocation_context: ApiInvocationContext):
```
Comment (bentsku) [inline localstack/services/apigateway/integration.py:123]: nit: I'm not too fond of mixing websockets logic in community as we've talked offline, but I believe for now, it's the quickest way, as it's already done for kinesis anyway. We will refactor this at some point!

**Review by bentsku — COMMENTED** [review]
Thanks a lot for tackling this! I believe there's a typo in the render logic, we render it to pass as the `template_key`. See the comments. Thanks!

File: localstack/services/apigateway/integration.py (hunk 1 above)
Comment (calvernaz) [reply localstack/services/apigateway/integration.py:703]: no this is correct, the template_key is rendered and becomes the request template

File: localstack/services/apigateway/integration.py (hunk 2 above)
Comment (calvernaz) [reply localstack/services/apigateway/integration.py:488]: yes, can be done on this PR as well.

File: localstack/services/apigateway/integration.py (hunk 1 above)
Comment (bentsku) [reply localstack/services/apigateway/integration.py:703]: Ah my bad! I see it now! Thanks a lot 👍

**Review by bentsku — APPROVED** [review]
LGTM! Sorry for the wrong comment.

**Pull request merged**